consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/api/IClaimStatusUpdate.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.moj.sdt.cmc.consumers.api;
+
+import uk.gov.moj.sdt.cmc.consumers.request.ClaimStatusUpdateRequest;
+import uk.gov.moj.sdt.cmc.consumers.response.ClaimStatusUpdateResponse;
+
+public interface IClaimStatusUpdate {
```

Comment:
To avoid confusion with the request type and make it more specific, rename this interface to ```IClaimStatusUpdateService```

Reply:
done